src/server/sched_func.c

```diff
@@ -439,9 +439,11 @@ set_sched_default(pbs_sched* psched)
 	}
 	if ((psched->sch_attr[(int) SCHED_ATR_sched_state].at_flags & ATR_VFLAG_SET) == 0) {
 		psched->sch_attr[(int) SCHED_ATR_sched_state].at_val.at_str = malloc(SC_STATUS_LEN + 1);
-		if (psched->sch_attr[(int) SCHED_ATR_sched_state].at_val.at_str == NULL)
+		if (psched->sch_attr[(int) SCHED_ATR_sched_state].at_val.at_str == NULL) {
+			snprintf(log_buffer, LOG_BUF_SIZE-1, "malloc failure (errno %d)", errno);
```

Comment:
You don't need the -1, snprintf() will handle that for you.  Same goes for the other snprintf() calls.

Reply:
Done